dmn-pmml-quarkus-example/src/test/java/org/kie/dmnpmml/kogito/quarkus/example/CommonTestUtils.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kie.pmml.kogito.quarkus.example;
+package org.kie.dmnpmml.kogito.quarkus.example;
```

Comment:
What about `dmn.pmml` here and also on other places?
```suggestion
package org.kie.dmn.pmml.kogito.quarkus.example;
```

Reply:
+1

Reply:
done as https://github.com/kiegroup/kogito-examples/pull/775/commits/5f47949ca2cc79cbc8d7fea87c5b34e556f80e72